### ISSUE TITLE
Optionally support the host option

### ISSUE THREAD

**Issue by fauno** [hyphacoop/api.distributed.press#33]
To run swagger-codegen-cli from Docker I needed #31 to bind to any address (`::`), I did this on JS:

```diff
diff --git a/v1/index.ts b/v1/index.ts
index 3eba15a..4f48053 100644
--- a/v1/index.ts
+++ b/v1/index.ts
@@ -1,8 +1,9 @@
 import apiBuilder from './api/index.js'
 
 const PORT = Number(process.env.PORT ?? '8080')
+const HOST = process.env.HOST ?? '::1';
 const server = await apiBuilder({ useLogging: true, useSwagger: true, usePrometheus: true })
-server.listen({ port: PORT }, (err, _address) => {
+server.listen({ port: PORT, host: HOST }, (err, _address) => {
   if (err != null) {
     server.log.error(err)
     process.exit(1)
```

But it's not the same as not sending the host option.

**Comment by fauno** [comment]
@jackyzha0

**Comment by jackyzha0** [comment]
Closed in e055b09
